### PR TITLE
Mark advanced settings as advanced in settings

### PR DESCRIPTION
**Pull request by MaxLeiter**
screen shot 2018-03-25 at 11 09 52 am

### CODE REVIEW

File: client/css/style.css [hunk 1]
```diff
@@ -1598,6 +1598,12 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 	display: none;
 }
 
+#settings [data-advanced] h2::after {
+	content: "advanced";
```
Comment (xPaw) [inline client/css/style.css]: That's gonna be painful to translate and you know it :)

Comment (MaxLeiter) [reply client/css/style.css]: translations be damned?

Comment (MaxLeiter) [reply client/css/style.css]: ill find another way 😞

**Comment by xPaw** [comment]
@MaxLeiter Why not just put it in the options template?

**Comment by MaxLeiter** [comment]
@xPaw: feels like that's duplicating the fact an option is `advanced`. Also, plugins should be able to add advanced options long-term and this will just complicate the template. I'm open to changing it.

**Pull request closed without merging**